_health-care/_js/actions/index.js

```diff
@@ -1,5 +1,7 @@
 export const ENSURE_FIELDS_INITIALIZED = 'ENSURE_FIELDS_INITIALIZED';
 export const VETERAN_FIELD_UPDATE = 'VETERAN_FIELD_UPDATE';
+export const SECTION_COMPLETE_UPDATE_FIELD = 'SECTION_COMPLETE_UPDATE_FIELD';
```

Comment:
The names for these actions are a bit unclear. `VETERAN_FIELD_UPDATE` is also a bit unclear, which is not from this PR. I like the convention of starting an action name with a verb, showing that it is an action (like `ENSURE_FIELDS_INITIALIZED` does). What if we name these `UPDATE_COMPLETION_STATUS` and `UPDATE_REVIEW_STATUS` instead?